locales/en.default.schema.json

```diff
@@ -863,7 +876,8 @@
           "label": "Show text below images"
         },
         "adapt_height_first_image": {
-          "label": "Adapt section height to first image size"
+          "label": "Adapt section height to first image size",
+          "info": "Overwrites image banner height setting when checked."
```

Comment:
We'll keep this for now but we know that moving this setting inside the `Image banner height` setting would be a better enhancement for merchants but would require a breaking change. Took a note [about it over here](https://github.com/Shopify/dawn/issues/697) to consider later.